doc/content/source/functions/BaselineCorrection.md

```diff
@@ -0,0 +1,409 @@
+
+## Usage id=usage
+
+The `BaselineCorrection` object requires, as input, some sort of record of an acceleration time history. This may be provided by specifying the `data_file` parameter, as demonstrated in [example-1], or, directly, by specifying the `time_values` and `acceleration_values` parameters, as demonstrated in [example-2].
+
+!listing baselinecorrection/twenty_sines.i
+         block=Functions
+         id=example-1
+         caption=Sample input syntax for `BaselineCorrection` with the `data_file` parameter used to input a twenty-cycle sinusoidal acceleration time history.
+
+!listing baselinecorrection/half_sine.i
+         block=Functions
+         id=example-2
+         caption=Sample input syntax for `BaselineCorrection` with the `time_values` and `acceleration_values` parameters used to input a half-sine acceleration time history.
+
+The `beta` and `gamma` parameters are those used for Newmark Integration. Ideally, they should correspond to the values used elsewhere throughout the input file. The `accel_fit_order`, `vel_fit_order`, and `disp_fit_order` variables control whether or not to use the polynomials present in [corrected-accel] when formulating the corrected time histories. If a value is specified, then a polynomial of that type and order will be found. Note that these are the parameters that define which of the combinations, described in [correction-combos], is used. For example, the set of inputs shown in [example-1] produces combination +AVD+, while those shown in [example-2] produces combination +A+.
+
+The order of the approximating polynomials, $n$, which are set by the `accel_fit_order`, `vel_fit_order`, or `disp_fit_order` parameters, can be assigned a maximum integer value of nine, since the computations typically become unstable when attempting to find polynomials of orders greater than nine. The corrections are highly sensitive to the approximation order, and so it is recommended that a user run with several trial values until the desired result is achieved. In general, the order should be less than that which is required to produce a true best-fit curve of $\ddot{u}(t)$. For example, in [example-2], a half-sine wave was input. Since a half-sine wave could be, nearly, perfectly approximated by a certain quadratic function, the order for the corrections was set to one. However, for higher order inputs, such as the 20-cycle sine wave used in [example-2], finding the right correction configuration becomes tricky, and is a matter of user-preference. The [#demo] section provides several examples that should clarify why it is important to choose a suitable polynomial order.
+
+!alert tip title=Use the Fewest Polynomials with the Lowest Orders Possible
```

Comment:
Can you please describe what happens if a user does `overcorrection`?

Reply:
@somu15 yes